runbook/executor.go

```diff
@@ -45,7 +65,7 @@ func Execute(executablePath string, args, environmentVars []string, stdout, stde
 		cmd = exec.Command(executablePath, args...)
 	}
 
-	cmd.Env = append(os.Environ(), environmentVars...)
+	cmd.Env = append(append(os.Environ(), fmt.Sprintf("JEC_CALLBACK_PIPE_PATH=%s", pipePath)), environmentVars...)
```

Comment:
won't JEC_CALLBACK_PIPE_PATH be overriden by each worker concurrently? we have multiple workers running at the same time. better to add it into **args**